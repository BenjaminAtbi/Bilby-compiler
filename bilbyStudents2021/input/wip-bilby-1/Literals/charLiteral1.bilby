main {
	print #s $n;
	print #101 $n;
	print ### $n;
	print ##3 $n;
	print ##9 $n;
	print #s $s #101 $s ### $s ##3 $n;
}