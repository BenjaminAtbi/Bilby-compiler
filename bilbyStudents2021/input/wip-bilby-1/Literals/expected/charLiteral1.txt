s
A
#
3
9
s A # 3
